CSkyL.csproj

```diff
@@ -11,6 +11,11 @@
     <AssemblyName>CSkyL</AssemblyName>
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <AssemblySearchPaths>
+      $(AssemblySearchPaths);
+      C:\Program Files (x86)\Steam\steamapps\common\Cities_Skylines\Cities_Data\Managed\;
+      ..\..\..\..\..\SteamLibrary\steamapps\common\Cities_Skylines\Cities_Data\Managed\
+    </AssemblySearchPaths>
```

Comment:
The modification of this project file somehow messed up my building process.

I've tried `git checkout main -- CSkyL.csproj` and simply put this `AssemblySearchPaths` part in and rebuild. These are the warnings/errors:

![image](https://user-images.githubusercontent.com/14138331/171266287-045ad4d9-c9f7-49bb-9372-dbe5e50c4307.png)

Do you know what happened here? cause I have no clues why simply inserting these lines caused those errors/warnings to be generated.

Reply:
ok fixed it.